### PR TITLE
provision script

### PR DESCRIPTION
**Pull request by AntonStasheuski**
# Task

Create provisioning script for vagrant(Centos 8.3). Instance should have static IP for SSH connection.

1. Provision script(hw8_vagrant_provision.sh) should do following actions:
    * Create service user hw8. Home folder = /opt/hw8. Add ssh public key for           this user.
    * Add this user to sudoers group(sudo without password)
    * Update /etc/hosts file with record “127.0.0.1 myownapp.com”
    * Restrict ssh with access via password
    * Restrict ssh root access
    * Install mc, vim and git packages
    * Update DNS to 8.8.8.8 and 1.1.1.1
    * Check that address myownapp.com can be resolved. If not execute step 3
    * Print external(White) IP of you vagrant centos instance
    * Create dir /var/log/myownapp and set up hw8 user as owner of the dir
2. Learn rsync command
3. Refresh your knowledge regarding useful bash commands and 12 factors
4. How to put iftop command output to the file?

# Description

In general terms figured out the work with Vagrant. Write provision script and vagrant config.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Images

Screenshot 2021-02-21 at 15 58 44

### CODE REVIEW

File: vagrant_bash_3/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "ubuntu/trusty64"
```
Comment (mavrik90) [inline vagrant_bash_3/Vagrantfile]: Please read task list and choose appropriate OS

File: vagrant_bash_3/hw9_vagrant_provision.sh [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+RESOLV_CONF="/etc/resolv.conf"
+DNS_1="8.8.8.8"
+DNS_2="1.1.1.1"
+MY_APP="myownapp.com"
+CREATION_DIR_PATH="/var/log/myownapp"
+DEFAULT_GROUP="root"
+
+function add_user {
+  echo "Start create user: $USER_NAME"
+  sudo useradd -s /bin/bash -d ${HOME_DIR}/${USER_NAME} -m -G sudo ${USER_NAME}
```
Comment (mavrik90) [inline vagrant_bash_3/hw9_vagrant_provision.sh]: Adding to sudo 2 times

**Comment by mavrik90** [comment]
@AntonStashevski can I have output of the run?

File: vagrant_bash_3/Vagrantfile (hunk 1 above)
Comment (AntonStasheuski) [reply vagrant_bash_3/Vagrantfile]: move to centos

File: vagrant_bash_3/hw9_vagrant_provision.sh (hunk 2 above)
Comment (AntonStasheuski) [reply vagrant_bash_3/hw9_vagrant_provision.sh]: delete second sudo

**Comment by mavrik90** [comment]
Could you please put your pic to READM.md file?

**Pull request merged**